README.md

```diff
@@ -1,6 +1,15 @@
 # vinch
 
-An idea creation app.
+[![Build Status](https://travis-ci.org/getvinch/vinch.svg?branch=master)](https://travis-ci.org/getvinch/vinch)
+[![Coverage Status](https://coveralls.io/repos/github/getvinch/vinch/badge.svg?branch=ci-travis)](https://coveralls.io/github/getvinch/vinch?branch=ci-travis)
+[![Dependency status](https://david-dm.org/getvinch/vinch.svg)](https://david-dm.org/getvinch/vinch)
+
+Vinch is an app for ideation.
```

Comment:
Nitpick: Should we keep this lowercase to keep it consistent?

Reply:
Going with capitalization as it has a more business feel